src/utils/featheryClient/integrationClient.ts

```diff
@@ -319,12 +319,17 @@ export default class IntegrationClient {
     const { userId } = initInfo();
     const signer = fieldValues[action.envelope_signer_field_key];
     const payload: Record<string, any> = {
+      form_key: this.formKey,
       fuser_key: userId,
       documents: action.documents ?? [],
-      signer_email: signer
+      signer_email: signer,
+      use_quik: action.quik_documents
     };
-    if (action.quik_documents)
-      payload.quik = fieldValues[action.quik_json_field_key];
+
+    if (action.quik_tags_field_key) {
+      payload.quik_tags = fieldValues[action.quik_tags_field_key];
```

Comment:
let's add logic to parse the quik tags depending on the type of the field value that's being fetched here. i left the specific details in a comment on your backend PR